.editorconfig

```diff
@@ -0,0 +1,22 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+indent_size = unset
+
+[*.{json,css}]
+indent_size = 2
+
+[*.{pug,jade}]
```

Comment:
I'm not super familiar with EditorConfig syntax but I assume you are, and that this glob-like thing works?

Reply:
I just checked and it's literally on the homepage; I should stop being so lazy